src/codegen/llvm.zig

```diff
@@ -11786,6 +11786,7 @@ fn backendSupportsF16(target: std.Target) bool {
         .mipsel,
         .mips64,
         .mips64el,
+        .loongarch64, // FIXME: See: https://github.com/llvm/llvm-project/issues/93894
```

Comment:
I expect that we'll be upgrading to LLVM 19 in this release cycle, assuming there are no issues blocking the upgrade. So I'm not sure it's worth it to apply workarounds for issues we know will be fixed in 19.

Reply:
OK, remove it.